manifests/plugin/abrt/params.pp

```diff
@@ -1,6 +1,7 @@
 # Default parameters for the Pulp smart proxy plugin
 class foreman_proxy::plugin::abrt::params {
   $enabled                 = true
+  $version                 = 'installed'
```

Comment:
No, I was suggesting setting it ```$version``` in ```abrt::params``` to ```undef``` so puppet will fall back to whatever ```foreman_proxy::plugin``` has as default for ```$version```.